### PR TITLE
Event specific announcements

### PR DESCRIPTION
**Pull request by wwohlers**

### CODE REVIEW

File: src/views/ProfileView.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
         </router-link>
       </div>
       <div class="announcements-list__container">
-        <announcements-list />
+        <announcements-list sitewide count="10"/>
```
Comment (jackblanc) [inline src/views/ProfileView.vue:27]: what is "10"?

Comment (jackblanc) [reply src/views/ProfileView.vue:27]: should be a constant in utils, whatever it is.

Comment (wwohlers) [reply src/views/ProfileView.vue:27]: 10 is the number of announcements to display. I'll make it a constant

**Comment by jackblanc** [comment]
Hey, can you add the constant and resolve merge conflicts here? Let me know if there are any questions with the single event view merge conflicts.

**Pull request merged**